src/Pelagos/Bundle/AppBundle/Resources/views/Search/default.html.twig

```diff
@@ -3,19 +3,44 @@
 {% block head %}
     {{ add_css (
```

Comment:
By overwriting the head, you basically remove all the head stuff from the template.

Just use stylesheet and javascipts blocks to add stuff, prefix each block with {{ parent() }}

Reply:
I forgot, this is not the new framework yet.